spec/scoreCountersSpec.js

```diff
@@ -0,0 +1,18 @@
+// 'use strict';
+//
+// describe('ScoreCounters', function(){
+//   var scorecounters;
+//   var frame;
+//
+//   beforeEach(function(){
+//     frame = new Frame();
+//     scorecounters = new ScoreCounters();
+//   });
+//
+//   it('adds the total regular points up', function(){
+//     frame.rollOne(2);
+//     frame.rollTwo(2);
+//     expect(scorecounters.currentTotalRegularPoints()).toEqual(4);
```

Comment:
Nice! This code is testing the end user behaviour of the class 🙌  Testing all of the scenarios outlined in the readme would allow you to TDD code to cover the edge cases